Contributor Documentation/LSP Extensions.md

```diff
@@ -506,6 +506,102 @@ export interface SetOptionsParams {
 }
 ```
 
+## `workspace/_sourceKitOptions`
+
+New request from the client to the server to retrieve the compiler arguments that SourceKit-LSP uses to process the document.
+
+This request does not require the document to be opened in SourceKit-LSP. This is also why it has the `workspace/` instead of the `textDocument/` prefix.
+
+> [!IMPORTANT]
+> This request is experimental, guarded behind the `sourcekit-options-request` experimental feature, and may be modified or removed in future versions of SourceKit-LSP without notice. Do not rely on it.
+
+
+- params: `TextDocumentSourceKitOptionsRequest`
+- result: `TextDocumentSourceKitOptionsResult`
```

Comment:
Shouldn't these be named `SourceKitOptionsRequest` & `SourceKitOptionsResponse`?

Reply:
I guess it technically doesn’t matter because `LSP Extensions.md` is self-consistent but changed it.